lib/v2/changba/user.js

```diff
@@ -0,0 +1,86 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const { art } = require('@/utils/render');
+const path = require('path');
+const headers = { 'User-Agent': 'Mozilla/5.0 (iPhone; CPU iPhone OS 11_0 like Mac OS X) AppleWebKit/604.1.38 (KHTML, like Gecko) Version/11.0 Mobile/15A372 Safari/604.1' };
```

Comment:
RSSHub's UA can also be changed via envs.
https://github.com/DIYgod/RSSHub/blob/bdb0e125d28ee4048418ded3ccd083feedbb326e/lib/config.js#L40

Consider using that value if envs.UA is set.